praw/models/reddit/subreddit.py

```diff
@@ -733,9 +733,19 @@ def distinguish(self, thing, how='yes'):
             moderator level distinguish. 'no' removes any distinction. 'admin'
             and 'special' require special user priviliges to use.
 
+        :param sticky: Comment is stickied if True, placing it at the top of
+            the comment page regardless of score. If thing is not a top-level
+            comment, this parameter is silently ignored.
         """
+        data = {'how': how, 'id': thing.fullname}
+        if sticky:
```

Comment:
Can you do something like `if sticky and is_comment` so that the `sticky` attribute is only added in such cases?